### PR TITLE
feat: added the toast component

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

File: pages/components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@import url(/css/main.css);
+
+.toast-preview-container {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    justify-content: center;
```
Comment (bhtibrewal) [inline pages/components/Toast/toast.css]: not needed, refactor unnecessary code

**Review by bhtibrewal — APPROVED** [review]
Looks okay to me, good to merge

File: pages/components/Toast/toast.css (hunk 1 above)
Comment (pradipgharge) [reply pages/components/Toast/toast.css]: made the changes.

File: pages/components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    gap: 1rem;
+}
+
+.toast{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (ShekharDhangar) [inline pages/components/Toast/toast.css:16]: You could use flex as helper class and it will avoid repeating code

**Review by ShekharDhangar — APPROVED** [review]
Try to do the suggested changes. rest looks good

File: pages/components/Toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    background-color: var(--secondary-color-dark);
+    color: var(--primary-color);
+}
+
+.toast-info .btn-toast{
+    color: var(--primary-dark--color);
+    font-weight: 500;
+}
+
+.toast-whitebg{
```
Comment (gauravsinhaweb) [inline pages/components/Toast/toast.css]: Add a hyphen btw `toast-white-bg`. Apply this for all.

**Review by gauravsinhaweb — APPROVED** [review]
Nice work!, Good to merge after changes.

File: pages/components/Toast/toast.css (hunk 2 above)
Comment (pradipgharge) [reply pages/components/Toast/toast.css:16]: Thanks for the suggestion, that would add one more class to the element and increase specificity so keeping this way.

File: pages/components/Toast/toast.css (hunk 3 above)
Comment (pradipgharge) [reply pages/components/Toast/toast.css]: Thanks for the suggestion.

File: pages/components/Alerts/alerts.html [hunk 4]
```diff
@@ -76,7 +76,7 @@
                         <a href="/pages/components/Typography/typography.html">Typography</a>
                     </li>
                     <li class="sidebar-list-item">
-                        <a href="#">Toast</a>
```
Comment (dev-ashishsethi) [inline pages/components/Alerts/alerts.html:79]: you could have made whole li a link because now I have to specifically hover over the text for navigating

**Review by dev-ashishsethi — COMMENTED** [review]
looks good

**Pull request merged**